libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -268,3 +274,37 @@ reflexivity =
                 , NamePart `{reflexivity}
                 , TextPart "is not applicable."
                 ]
+
+||| Attempt to swap the sides of equality in a goal.
+symmetry : Elab ()
+symmetry = do
+    [_,_,_,_,_] <- apply (Var `{{sym}}) [True, True, True, True, False]
+       | _ => fail [TextPart "Failed while applying", NamePart `{symmetry} ]
+    solve
+    attack
```

Comment:
`attack` introduces a guess not a whole. 
I think you can make a hole yourself and plug it in with `exact` or `fill` instead (maybe that is a better idea, than using `solve`-`attack`).